x-pack/plugins/security_solution/public/assistant/stack_management/management_settings.tsx

```diff
@@ -34,8 +37,23 @@ export const ManagementSettings = React.memo(() => {
         securitySolutionAssistant: { 'ai-assistant': securityAIAssistantEnabled },
       },
     },
+    security,
   } = useKibana().services;
 
+  const { data: currentUserAvatar } = useQuery({
```

Comment:
This is usually [passed into the assistant](https://github.com/elastic/kibana/blob/27ccd4d5399a1b4d8b79037b86560ae73e77984e/x-pack/plugins/security_solution/public/assistant/overlay.tsx#L19-L30), so doing it manually here on the `management` side. 

Will probably need to extend (or get service off useKibana?) as we'll need to get any user's avatar for global entries.